### PR TITLE
Add Loping.net block explorers, add TestNet4 / ScaleNet explorer list

### PR DESCRIPTION
**Pull request by EchterAgo**

### CODE REVIEW

File: electroncash/web.py [hunk 1]
```diff
@@ -98,9 +125,14 @@ def BE_tuple(config):
            )
 
 def BE_default_explorer():
-    return (DEFAULT_EXPLORER
-            if not networks.net.TESTNET
-            else DEFAULT_EXPLORER_TESTNET)
+    netname = networks.net.__name__
```
Comment (cculianu) [inline electroncash/web.py:128]: Hmm. Would you mind terribly if I asked you to rewrite this to use:

```python
net = networks.net
if net is networks.TestNet:
     # stuff
elif net is networks.TestNet4:
    # otherstuff
# and so on...
```

`is` in Python tests if the object is the same exact object (in memory).... I feel that's more explicit .... even though probably equivalent to what you have here.  Just feels like it is "safer" and ore likely for an IDE to catch typos, etc.

File: electroncash/web.py [hunk 2]
```diff
@@ -82,13 +85,37 @@
...
 }
 
 def BE_info():
-    if networks.net.TESTNET:
+    netname = networks.net.__name__
+    if netname == 'TestNet':
         return testnet_block_explorers
+    elif netname == 'TestNet4':
+        return testnet4_block_explorers
+    elif netname == 'ScaleNet':
```
Comment (cculianu) [inline electroncash/web.py:117]: Here too.  See my other comment about `net is networks.ScaleNet`, etc..

File: electroncash/web.py (hunk 1 above)
Comment (EchterAgo) [reply electroncash/web.py:128]: Yea, that seems much better, will update.

**Comment by EchterAgo** [comment]
Updated

**Comment by cculianu** [comment]
Looks good  to me.  Tested, works.  Thanks for the update!

**Pull request merged**